README.md

```diff
@@ -330,753 +277,454 @@ Rails.application.routes.draw do
-
-When we use the `new` method, Rails is going to attempt to create an object with attributes that match up to column names that exist in a `tasks` table in our database; and, when we attempt to `save` that object, Rails will try to `INSERT` a new record in our database with those attributes. Can you see where we might run into problems? We haven't set up our database yet, much less any tables in that database!
-
-### Reviewing Our Database Set-up
-
-At the start of this tutorial we set up our database. Let's review what we did.
-
-First, we created our database with this command:
-
-```bash
-$ rails db:create
-```
-
-And got this output:
-
-```bash
-Created database 'task_manager_development'
-Created database 'task_manager_test'
-```
-
-We created two databases, a "development" and a "test" database. For this tutorial we are working in the "development" environment because we are not writing tests.
-
-We then created a migration with the instructions for creating a tasks table in our database by running the following command:
-
-```bash
-$ rails generate migration CreateTask title:string description:string
-```
-
-In this command, we told rails to generate a migration file that will create a tasks table in our database with two columns - title and description. You can take a look at your migration in a file named something like `db/migrate/20190414173402_create_task.rb`.
-
-We then ran the following code to apply our migration and actually create a database.
-
-```bash
-$ rails db:migrate
-```
-
-Let's review what our table currently looks like. In your terminal, connect to the database, and see what's currently in your task table.
-
-```bash
-$ rails dbconsole
-```
-
-```bash
-psql (14.2)
-Type "help" for help.
+class TasksController < ApplicationController
+  def index
+    render json: Task.All
+  end
 
-task_manager_development=# SELECT * FROM tasks;
- id | title | description | created_at | updated_at
-----+-------+-------------+------------+------------
-(0 rows)
+  def show
+    render json: Task.find(params[:id])
+  end
 
-task_manager_development=#
+  def create
+    task = Task.new({
+      title: params[:title],
+      description: params[:description]
+    })
+  
+    if task.save
+      render json: task, status: :created
+    else
+      render json: task.errors, status: :unprocessable_entity
+    end
+  end
+end
 ```
+<!-- TODO: do we want this error handling. Sort of seems like we want error handling of some sort here? -->
```

Comment:
Good question. I think because we're giving them step by step instructions it's okay to have. They can start to become familiar with the pattern even if they don't understand exactly what's happening. (I could also see an argument here that it's too much too soon - I don't feel super strongly either way).

Reply:
I think it might be too soon, and they won't necessarily see the benefit of extracting task.save if there are no data validations on the Task model (and there won't be at this point)

I'd vote for just doing a simple Task.create with those params, and not having an if statement here. They will learn about data validations later in the mod